### PR TITLE
[WIP] Muon Analysis example

### PR DESCRIPTION
**Pull request by mwinn2**
Adding a simple single loop prototype analysis for muon-arm data in analogy to simpletrackanalysis for O2, @ktf for checking.

### CODE REVIEW

**Comment by ktf** [comment]
This works fine for me actually and I get the correct plot. The important bit is that you do:

```bash
alienv enter O2/latest Run2ESDConverter/latest
```

when you launch it, because it needs the converter in the environment.

**Comment by mwinn2** [comment]
Hi @ktf ,
Thanks for fixing the typo and the explanations!
The log is at:
O2/Framework/TestWorkflows/log.txt
in this branch. In pricnciple, it just didn't fill, I used both packages.
I recompiled, I will write, if the problem persists with the new log if needed.

**Comment by mwinn2** [comment]
Hi @ktf,
ok, the problem is still there, I updated the log with the new run, but I think that it has the same problems.

**Comment by ktf** [comment]
Is that all? You should see a message "All data has been processed". Maybe you are `Ctrl-C`ing to early?

**Comment by mwinn2** [comment]
Ok, it is still running, I leave it and hopefully, it will be fine. If it is, it is however surprising that it takes so long.

**Comment by ktf** [comment]
To be clear, it should be just a few seconds, not minutes.

File: Framework/TestWorkflows/log.txt [hunk 1]
```diff
@@ -0,0 +1,256 @@
+alidock
```
Comment (sawenzel) [inline Framework/TestWorkflows/log.txt]: should this file be committed?

**Comment by ktf** [comment]
No, this is just to share information.

**Comment by mwinn2** [comment]
Hi @ktf,
for the record (I came only now back to it): it works now! Thanks a lot for the debugging.
Now that the basic functionality works, I will implement a few prototype ideas and will let you and Gian-Michele know.

**Comment by ktf** [comment]
thanks for letting me know. I think there is still some issue with how I convert the files, due to padding, but I am working on it.

**Comment by sawenzel** [comment]
What is the status of this?

**Comment by mwinn2** [comment]
@sawenzel ,
I can have a look beginning of next week. I didn't have time to come back to it. Maybe would be good to have an example in the code.

**Comment by sawenzel** [comment]
@mwinn2 @ktf: What should we do here? If this is still a useful example demonstration DPL features in analysis, a merge might be useful (or providing this code in a snippet or doc file). Should we close it otherwise and re-open when development finished?

**Comment by sawenzel** [comment]
@mwinn2 : I am closing this for the moment since there is no activity and to cleanup our list a bit.
Feel free to reopen any time.

**Pull request closed without merging**